File: csharp/ex038.cs
// Escreva um programa que mostre na tela a seguinte contagem:
// 6 7 8 9 10 11 Acabou!


class Ex038
{
  public int counter = 6;

  public void FirstCountdown()
  {
    while (counter <=11)
    {
      counter++;
      Console.Write(counter);
    }
    Console.Write("Acabou!");
  }
}

File: typescript/src/ex038.ts
// Escreva um programa que mostre na tela a seguinte contagem:
// 6 7 8 9 10 11 Acabou!


let numbValue : number = 5;
let resultado : string = "";

while (numbValue <= 10) {
    numbValue+=1;
    resultado += numbValue + " ";
}
console.log(`${resultado} Acabou!`);

